.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,18 +28,18 @@
 <!-- Does this change the generated Verilog or the simulator memory map of the default targets?  -->
 
 ### Contributor Checklist
-- [ ] Did you set dev as the base branch?
-- [ ] Did you add Scaladoc to every public function/method?
+- [ ] Is this PR's title suitable for inclusion in the changelog and have you added a `changelog:<topic>` label?
```

Comment:
I know you said it above, but maybe "If not, add the label changelog:omit"  in a secondary bullet?